### PR TITLE
fix(PageHeader): removing redundant role="banner" attribute

### LINKED ISSUE #3498: PageHeader: role=banner attribute on <header>
**Describe the issue. What is the expected and unexpected behavior?**

This issue is related to [#2580](https://github.com/patternfly/patternfly-react/issues/2580). 

The error I get from AXE is “Banner landmark must not be contained in another landmark”, and the element looks like this currently: `<header role="banner" class="pf-c-page__header">` .

According to the ARIA docs, the `role="banner"` doesn’t need to be there at all, which is where the error is coming from.  

"Assistive technologies can identify the main header element of a page as the banner if is a descendant of the body element". In this case, it _is_ a descendant of the body element, and therefore the `role="banner"` is redundant. (See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Banner_Role).

It looks like the folks at Jekyll have addressed a similar issue:
https://github.com/jekyll/minima/issues/75

### PR DESCRIPTION
**Pull request by jenny-s51**
Closes #3498.

**Additional issues**:  Addresses #2580.

### CODE REVIEW

**Comment by tlabaj** [comment]
@jenny-s51 This will not close #2580 correct?

**Comment by jenny-s51** [comment]
@tlabaj yes that's correct -- that issue involves the `main` attribute of `Page` as well. I'm happy to remove that attribute so that issue can also be closed! It was out of scope of this PR, but after looking into it, seems like we can just remove that attribute without running into any problems. Would you like me to do that? Should be a very quick fix 🙂

**Comment by tlabaj** [comment]
@jenny-s51 Yes, can you please do that.  Thanks!

**Comment by jenny-s51** [comment]
@tlabaj yes! checking with @seanforyou23 now to see if doing that would resolve the issue in the way he hoped

File: packages/patternfly-4/react-core/src/components/Page/PageHeader.tsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ export const PageHeader = ({
         const navOpen = isManagedSidebar ? managedIsNavOpen : isNavOpen;
 
         return (
-          <header role="banner" className={css(styles.pageHeader, className)} {...props}>
+          <header className={css(styles.pageHeader, className)} {...props}>
```
Comment (seanforyou23) [inline packages/patternfly-4/react-core/src/components/Page/PageHeader.tsx]: Can we allow this attribute to be omitted by default, but set if the user provides a value explicitly?

Comment (jenny-s51) [reply packages/patternfly-4/react-core/src/components/Page/PageHeader.tsx]: Absolutely! Will open up a separate issue for adding a new prop that will do this.

Comment (seanforyou23) [reply packages/patternfly-4/react-core/src/components/Page/PageHeader.tsx]: Ty!

**Pull request merged**